yarn.lock

```diff
@@ -2272,9 +2272,9 @@
   integrity sha512-jhuKLIRrhvCPLqwPcx6INqmKeiA5EWrsCOPhrlFSrbrmU4ZMPjj5Ul/oLCMDO98XRUIwVm78xICz4EPCektzeQ==
 
 "@types/node@*", "@types/node@>=14.0":
-  version "18.7.23"
-  resolved "https://registry.yarnpkg.com/@types/node/-/node-18.7.23.tgz#75c580983846181ebe5f4abc40fe9dfb2d65665f"
-  integrity sha512-DWNcCHolDq0ZKGizjx2DZjR/PqsYwAcYUJmfMWqtVU2MBMG5Mo+xFZrhGId5r/O5HOuMPyQEcM6KUBp5lBZZBg==
+  version "18.11.18"
```

Comment:
I didn't realize we were using node 18 types. I would feel safer using node 16 types as that is more representative of the platform we support.

Reply:
True. Though IMO we can rely on the unit tests that run on 14, 15 and 18 for platform compatibility.

Regardless, out of scope of this PR but feel free to make one for that.